marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/DeployHubDefaultTdesCommand.java

```diff
@@ -0,0 +1,66 @@
+package com.marklogic.hub.deploy.commands;
+
+import com.marklogic.appdeployer.command.AbstractCommand;
+import com.marklogic.appdeployer.command.CommandContext;
+import com.marklogic.appdeployer.command.SortOrderConstants;
+import com.marklogic.client.document.DocumentWriteOperation;
+import com.marklogic.client.document.DocumentWriteSet;
+import com.marklogic.client.document.JSONDocumentManager;
+import com.marklogic.client.ext.util.DefaultDocumentPermissionsParser;
+import com.marklogic.client.ext.util.DocumentPermissionsParser;
+import com.marklogic.client.impl.DocumentWriteOperationImpl;
+import com.marklogic.client.io.DocumentMetadataHandle;
+import com.marklogic.client.io.StringHandle;
+import com.marklogic.hub.HubConfig;
+import com.marklogic.hub.impl.HubConfigImpl;
+import org.apache.commons.io.IOUtils;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
+import org.springframework.core.io.support.ResourcePatternResolver;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class DeployHubDefaultTdesCommand extends AbstractCommand {
```

Comment:
@rahulvudutala , @ryanjdew 
I'm not sure what the right name for the class . But if  'hubInstallTemplates' is going to be the command to load these templates (https://project.marklogic.com/jira/browse/DHFPROD-7052?focusedCommentId=197488&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-197488), can this be class be LoadHubTemplatesCommand or is this name too generic ?

Reply:
Deploy is the verb we current more consistently use in our tasks and commands. DeployHubDefaultTdesCommand makes sense to me. If we switch to install, we should probably do it at a time when we can make a larger change to be consistent.